content/api/_index.md

```diff
@@ -35,6 +35,19 @@ If this file is missing or incorrectly named, your robot won't do anything. No l
 ## Start Button
 After the robot has finished starting up, it will wait for the _Start Button_ on the power board to be pressed before continuing with your code, so that you can control when it starts moving. There is a green LED next to the start button which flashes when the robot is finished setting up and the start button can be pressed.
 
+### Doing things before pressing the start button
```

Comment:
I think `Running Code before pressing the start button` might be a better way to phrase this?